crowbar_framework/app/helpers/barclamp/suse_manager_client_helper.rb

```diff
@@ -25,7 +25,7 @@ def suse_manager_client_rpm
     end
 
     def suse_manager_client_install
-      "/opt/dell/bin/barclamp_install.rb --rpm suse-manager-client"
+      "knife cookbook upload core -o /opt/dell/chef/cookbooks"
```

Comment:
The lib is designed for such use cases, so it will figure out which cookbook is missing an will just upload this one.